test/integration/roles/test_openssl/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+- name: Generate a private key
```

Comment:
Install whatever dependencies are needed to the OS listed in 
https://github.com/ansible/ansible/blob/devel/test/runner/completion/docker.txt
+ 

freebsd/10.3-STABLE
freebsd/11.0-STABLE
osx/10.11
rhel/7.3